### ISSUE TITLE
CJ4 Autopilot not following flight plan in NAV mode

### ISSUE THREAD

**Issue by sergiolamares** [Working-Title-MSFS-Mods/fspackages#415]
suddenly while in cruise mode, AP stops following flight plan, and waypoint distance show's different in PFD comparing to MCDU Legs page

**Comment by dga711** [comment]
was the waypoint in question part of arrival/approach?

**Comment by sergiolamares** [comment]
No, it was mid flight, after reaching cruise level.

**Comment by dga711** [comment]
We were thinking about if there could be a problem with modifying the flight plan to a DirectTo. 
But since that seems to not be the case here, we would have to further investigate.

**Comment by cwburnett** [comment]
Pending FPM

**Comment by mikebakke** [comment]
fwiw I had this yesterday in the transition for approach to EGNT. I overflew the field at 3500 which should have then gone hard right over the sea for a final turn to finals but it just carried on and I had to hand fly back to a reasonable intercept. Localiser acquisition didn't really happen well, seemed to be about quarter mile to right of centerline and glideslope never took hold. Both localiser and slope were on the instruments but the a/p was just not doing it.

I know the approach well as I fly it a lot in the TBM and that is (now) rock solid. Not sure if it's because I imported the plan from simbrief rather than manually entering but will try to repeat plugging it in myself.

Regardless - really appreciate the work being done on this model and love seeing the evolution!

Thanks to all involved.
